includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -1195,6 +1199,23 @@ public function add_orderby_custom_field_to_query( WP_User_Query $query ) {
 		);
 	}
 
+	/**
+	 * Order query based on the custom field.
+	 *
+	 * @since  4.3.0
+	 * @access private
+	 *
+	 * @param array  $args Arguments Old orderby arguments.
+	 * @param object $query Query.
+	 */
+	public function add_orderby_custom_field_to_non_user_query( $args, $query ) {
+		return sprintf(
```

Comment:
There was some [previous advice](https://github.com/Automattic/sensei/pull/4960#discussion_r836872272) that should probably be applied here as well.

Reply:
yes, it was not working when I tried using it, but now I realised I've added "ORDER BY" in the string so that was the reason it was not working. 

I've changed it now: [commit](https://github.com/Automattic/sensei/pull/4965/commits/1d87c9a9bc4310679c4eb4387dd5b0d5e3a7d513)